### DISCUSSION TITLE
【分享】不用學程式也可以設計前端的工具 !

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#505] [Ideas]
### 前端 No Code 工具
有寫過網頁前端的同學應該知道前端的痛楚，在設計框架或某個元素的屬性有語法上的困擾，而我要推薦的 Liveweave 可以在**設計好後查看原始碼**，我認為這對那些不太懂 HTML CSS 語法的同學很友善，**甚至還可以邀請好友們一同編輯**

[https://liveweave.com/](url)

**Comment by manipulateed** [comment]
手寫html css 的確是很痛苦的事情，但這個能解決這個問題，而且也不會讓我們不知道背後原始碼是如何，所以我覺得算是很不錯的一個工具。

**Comment by NcuJerryKao** [comment]
前端固然重要 但一起討論 用非程式語言開發 也是很有效的

**Comment by williamyen0623** [comment]
現在很多開發項目都有圖形化介面的工具可以用，非常方便，可以省去不少時間，但缺點是有時候需要客製化可能會沒辦法辦到

**Comment by BingRong1217** [comment]
感覺是很好用的工具，有機會可以嘗試看看！

**Comment by xCk27x** [comment]
連結打不開~
應該是這個 https://liveweave.com/

**Comment by xCk27x** [comment]
剛剛試了一下 
對單一網頁面板的設計來說是很方便沒錯
但如果整個網頁的框架太複雜
就有點難追蹤程式碼了QQ

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Yilin20021016** [comment]
感覺這對我很友善 但是會有一些小小問題嗎

**Comment by bww0401** [comment]
天啊這對程式沒有很強的人實在太友善(我，而且我覺得一同編輯是一大賣點，像是很多小組作業，如果是需要用到程式開發，就會因為沒辦法共同編輯而必須面對面來討論，因此如果可以線上共同編輯，將是我這種懶人的一大福祉

**Comment by larry40305** [comment]
可惡，沒有在期末專題前看到這個。不然就不用網站寫那麼久了

**Comment by awkward-willy** [comment]
現在有各式各樣的 No-code / Low-code 工具，能方便人們更快速的完成自己所需的功能，但相對而言，工程師等職業就更須培養自己的專業技能、獨立思考與解決問題的能力，同時善用各式各樣這類型的工具，才能展現在職場上的價值。
